platform/o.n.swing.flatlaf/src/org/netbeans/swing/flatlaf/Installer.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.netbeans.swing.flatlaf;
+
+import javax.swing.UIManager;
+import org.openide.modules.ModuleInstall;
+
+public class Installer extends ModuleInstall {
+
+    @Override
+    public void validate() throws IllegalStateException {
+        UIManager.installLookAndFeel(new UIManager.LookAndFeelInfo("Flat Dark", NbFlatDarkLaf.class.getName()));
```

Comment:
If you're planning on merging this before it's fully done, maybe rename this "FlatLaf Dark (experimental)" so people don't assume it's meant to be used when they see it as an option in the Preferences pane.